<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultRecordPage</class>
 <widget class="QWidget" name="ResultRecordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_disciplineContents">
     <property name="text">
      <string>Выберите уточнение дисциплины</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_disciplineContents">
     <property name="placeholderText">
      <string>Уточнение дисциплины</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_result">
     <property name="text">
      <string>Введите результат</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le_result_1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="le_result_2"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
